dotnet/src/VectorDataIntegrationTests/AzureAISearchIntegrationTests/Filter/AzureAISearchBasicQueryTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using AzureAISearchIntegrationTests.Support;
+using VectorDataSpecificationTests.Filter;
+using VectorDataSpecificationTests.Support;
+using Xunit;
+
+namespace AzureAISearchIntegrationTests.Filter;
+
+public class AzureAISearchBasicQueryTests(AzureAISearchBasicQueryTests.Fixture fixture)
+    : BasicQueryTests<string>(fixture), IClassFixture<AzureAISearchBasicQueryTests.Fixture>
+{
+    // Azure AI Search only supports search.in() over strings
+    public override Task Contains_over_inline_int_array()
```

Comment:
This unfortunately forces us to duplicate connector-specific overrides/ignores etc. I think it's OK to merge like this for now, but I'll probably do a cleanup to just have the BasicFilterTests take care of covering it, this way we have a single test suite that covers both filtering as part of vector search, and not as part of vector search. In fact, since we also have the dynamic version of each filter as well (using dictionary key lookup), each test would have four variants (POCO vs dynamic, vector search vs. non-vector search).

I can take care of this later - I've already done the work for the dynamic side so I'll consolidate later.